documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,180 @@
+.. The `Artefact Id` and `Package` fields are automatically populated based on the parent project details.
++
+.Configure the Module Archetype Parameters
+image::mvnbasic_module_archetype_parameters.png[Configure the Module Archetype Parameters]
++
+.. Click `Finish` to conclude setting up the Maven module.
+. (Optional) To change the settings for the created Maven module, expand the module name in the `Project Explorer` view and double click `pom.xml` from the expanded list. An `Overview` tab appears and all settings can be changed from this tab.
++
+.Change the Module Settings from the Overview View
+image::mvnbasic_module_pom_overview.png[Change the Module Settings from the Overview View]
++
+
+**Result:** Your new Maven module is created and appears in the `Project Explorer` view. Additionally, a hierarchical view of the nested projects is now available in the Project Explorer view as well (see https://www.eclipse.org/eclipse/news/4.5/M5/#nested-projects[Nested/Hierarchical view of projects]).
+
+
+[[Maven_Support]]
+== 3. Add Maven Support to an Existing Non-Maven Project
+
+The previous tasks contain instructions to create a new Maven project and Maven module. However, for an existing application that was not created with Maven support, use the following instructions to add Maven support to the non-Maven project:
+
+==== Adding Maven Support to an Existing Non-Maven Project ====
+////
+. (Optional) To add dependencies to the project, use the following instructions:
+.. Right-click the project name in the `Project Explorer` view and click `Properties`.
+.. In the `Java Build Path` window, click `Libraries`.
++
+.Set the Java Build Path
+image::mvnbasic_java_build_path.png[Set the Java Build Path]
++
+.. Click `Add External JARs`, navigate to the required JAR file and select it. The new JAR will appear in the list in the `Libraries` tab.
+.. Click `OK` to confirm the changes.
+////
+. Right-click the project name in the `Project Explorer` view.
+. From the displayed options, click `Configure`.
+. From the displayed sub-menu, click `Convert to Maven Project`.
+. Configure details for the new pom file:
+.. The basic fields for the new pom file are prepopulated based on the project details. If required, edit the existing values.
+.. (Optional) Add a name for the new project in the `Name` field.
+.. (Optional) Add a brief description for the project in the `Description` field.
++
+.Create a New Pom Descriptor
+image::mvnbasic_create_new_pom.png[Create a New Pom Descriptor]
++
+.. Click `Finish` to finalize the pom information.
+. If dependencies were added (using the instructions in step 1), a wizard appears displaying all added dependencies and a green check mark when each dependency is identified. Learn more about dependency identification in the <<did_you_know,Did You Know>> section.
```

Comment:
There's no step1 instructions anymore. Rather "If the project references java dependencies, a wizard appears displaying all these dependencies ..."